### ISSUE TITLE
sift test fails

### ISSUE THREAD

**Issue by mretegan** [silx-kit/silx#274]
Python 3.5, OS X 10.11
```
======================================================================
ERROR: test_transform (silx.image.sift.test.test_transform.TestTransform)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/marius/Software/silx/build/lib.macosx-10.11-x86_64-3.5/silx/image/sift/test/test_transform.py", line 188, in test_transform
    sol, MSE = self.matching_correction(image, image2)
  File "/Users/marius/Software/silx/build/lib.macosx-10.11-x86_64-3.5/silx/image/sift/test/test_transform.py", line 124, in matching_correction
    kp1 = s.keypoints(image)
  File "/Users/marius/Software/silx/build/lib.macosx-10.11-x86_64-3.5/silx/image/sift/plan.py", line 507, in keypoints
    kp, descriptor = self._one_octave(octave)
  File "/Users/marius/Software/silx/build/lib.macosx-10.11-x86_64-3.5/silx/image/sift/plan.py", line 669, in _one_octave
    wgsize2 = self.kernels[file_to_use]
KeyError: 'keypoints_gpu2'
```

**Comment by payno** [comment]
Should be fixed by #303

**Comment by mretegan** [comment]
Yes, the test passes now.
